device/dhcp/common/dhcpv6options.yaml

```diff
@@ -0,0 +1,387 @@
+
+    Dhcpv6Options.IaAddress:
+      description: >-
+        The IA Address option is used to specify IPv6 addresses associated with an IA_NA or an IA_TA. The IA Address option must be 
+        encapsulated in the Options field of an IA_NA or IA_TA option.
+      type: object
+      properties:
+        ip:
+          description: >-
+            The IPv6 address associated with an IA_NA or an IA_TA.
+          type: string
+          format: ipv6
+          x-field-uid: 1
+        preferred_lifetime: 
+          description: >-
+            The preferred lifetime for the IPv6 address in expressed in units of seconds.
+          type: integer
+          format: uint32
+          x-field-uid: 2
+        valid_lifetime: 
+          description: >-
+            The valid lifetime for the IPv6 address in expressed in units of seconds.
+          type: integer
+          format: uint32
+          x-field-uid: 3
+
+    Dhcpv6Options.OptionsRequest:
+      type: object
+      properties:
+        choice:
+          description: >-
+            The Option Request option is used to identify a list of options in a message between a client and a server.
+            - Vendor_specific information option, requested by clients for vendor-specific informations from servers.
+            - DNS Recursive Name Server Option, requested by clients to get the list ofIPv6 addresses of DNS recursive name 
+              servers to which DNS queries may be sent by the client resolver in order of preference.
+            - Client FQDN option - indicates whether the client or the DHCP server should update DNS with the AAAA record 
+              corresponding to the assigned IPv6 address and the FQDN provided in this option. The DHCP server always updates 
+              the PTR record.
+            - bootfile_url, URLvto a boot file.
+            - sztp. Securely provision a networking device when it is booting in a factory-default state.
+          type: string
+          default: vendor_information
+          x-field-uid: 1
+          x-enum:
+            vendor_information:
+              x-field-uid: 1
+            name_servers:
+              x-field-uid: 2
+            fqdn:
+              x-field-uid: 3
+            bootfile_url:
+              x-field-uid: 4
+            sztp:
+              x-field-uid: 5
+            custom:
+              x-field-uid: 6
+        custom:
+          $ref: '#/components/schemas/Dhcpv6Options.Custom'
+          x-field-uid: 2
+
+    Dhcpv6Options.Custom:
+      description: >-
+        The Custom option is used to provide a not well known option in the message between a client and a server.
+      type: object
+      properties:
+        type:
+          description: >-
+            The type of the Custom option TLV.
+          type: integer
+          format: uint32
+          x-field-uid: 1  
+        value:
+          description: >-
+            The value of the Custom option TLV.
+          type: string
```

Comment:
Should this type be string or hex . If yes , is it expected to be padded to byte boundary ?

Reply:
changed to integer

Reply:
Not able to see the change.